contrib/examples/example.c

```diff
@@ -249,6 +249,26 @@ int main() {
             printf("Address: %s\n", str);
         }
 
+	// EXTENDED PUBLIC KEY DERIVATION EXAMPLE
+	printf("\n\nBEGIN EXTENDED PUBLIC KEY DERIVATION:\n\n");
+
+	// Generate the Master key from a seed
+	dogecoin_hdnode_from_seed(utils_hex_to_uint8("000102030405060708090a0b0c0d0e0f"), 16, &node);
+
+	// Serialize and print the Master public key
+	char master_public_key[112] = {0};
+	dogecoin_hdnode_serialize_public(&node, &dogecoin_chainparams_main, master_public_key, sizeof(master_public_key));
```

Comment:
Just going to keep remarking to keep it in mind: we need to do something differently than requiring memaddresses for pointers here (&node, &chainparams) - so we need to definitely say a TODO: on adding a different way to access these.  Other languages may not be able to access or manipulate pointer addresses; and we don't want to restrict. Probably for chainparams a way to select main or testnet;  node probably can be something that works out an internal reference.

Reply:
Agreed, I'm working on wrappers to pass individual parameters back by reference.
https://github.com/edtubbs/libdogecoin/commit/56f08a75d6067cc5de548961a814ee58a3d66f6f#diff-b6c5c7f340ac317dd7d04288fc653a288892e2fa73389a9ed8a073328a3858e7R237-R256

Reply:
Resolved in #158.